libbpf-rs/src/program.rs

```diff
@@ -376,6 +384,16 @@ impl Program {
         }
     }
 
+    /// Return true if the bpf program is set to autoload else false
```

Comment:
Consider using markdown for boolean values:
```patch
-/// Return true if the bpf program is set to autoload else false
+/// Return `true` if the bpf program is set to autoload, `false` otherwise
```